src/utils.ts

```diff
@@ -549,4 +551,23 @@ export function isNumber(num: number): boolean {
 
 export function toVoidP(p: Promise<any>): Promise<void> {
     return p.then(() => { });
+}
+
+export function calculateElapsedTime(startProcessingTime: [number, number]): number {
```

Comment:
Nit: Wouldn't it be better to type the tuple as "HighRes timer" or something similar?

Reply:
Sounds good. Added that.